client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -147,7 +148,7 @@ export default class ProductPurchaseFeaturesList extends Component {
 			sitePlans,
 		} = this.props;
 
-		const plan = find( sitePlans.data, isWpcomPremium );
+		const plan = find( sitePlans.data, isWpcomPersonal );
```

Comment:
Hmm, for no apparent reason. Maybe while I moved it out from `ProductPurchaseFeatures`, I forgot to connect it. Will do in another PR. Thanks!